[APM] Renaming helper functions about transaction aggegations (#143299)
elastic/kibana: x-pack/plugins/apm/dev_docs/apm_queries.md

}
```

# Transactions in service inventory page

> A new aggregated metric document introduced and holds latency and throughput metrics pivoted by `service.name+service.environment+transaction.type`

Reviewer: It won't stay "new" forever :)

```suggestion
Service metric is an aggregated metric document that holds latency and throughput metrics pivoted by `service.name + service.environment + transaction.type`
```

The decision to use service metrics aggregation or not is determined in [getServiceInventorySearchSource](https://github.com/elastic/kibana/blob/5d585ea375be551a169a0bea49b011819b9ac669/x-pack/plugins/apm/server/lib/helpers/get_service_inventory_search_source.ts#L12) and [getSearchServiceMetrics](https://github.com/elastic/kibana/blob/5d585ea375be551a169a0bea49b011819b9ac669/x-pack/plugins/apm/server/lib/helpers/service_metrics/index.ts#L38)

A pre-aggregated document where `_doc_count` is the number of transaction events

```
{
  "_doc_count": 2,
  "@timestamp": "2021-09-01T10:00:00.000Z",
  "processor.event": "metric",
  "metricset.name": "service",
  "service": {
    "environment": "production",
    "name": "web-go"
  },
  "transaction": {
    "duration.summary": {
        "sum": 376492831,

>         "value_count": 775

Reviewer: Should `value_count` and `_doc_count` be the same? 
cc @Mpdreamz

Author: YES! my fault copy-paste issue, should update it 

ref: https://github.com/elastic/apm-server/pull/9143/files
